### DISCUSSION TITLE
TIDAL Developer Portal API Go client

### DISCUSSION THREAD

**Discussion by tomjowitt** [tidal-music/discussions#46] [General]
Hi all,
I'm happy to see the TIDAL API coming along, and I appreciate you are opening up many things. Thanks for the great work.

I wrote some integration code a while back for a personal project as I couldn't find an SDK for Go and I thought I might as well turn it into a public Go package. It's still very light on functionality, but I'm adding endpoints frequently and should be up to parity with the TIDAL API fairly soon.

Code here:

https://github.com/tomjowitt/gotidal

**Comment by aliaksei-taliuk** [comment]
That's amazing! Thanks @tomjowitt for making your work publicly available! 
Stay tuned, even more exciting things are going to be opened soon.
